src/pages/Settings.js

```diff
@@ -261,48 +261,35 @@ export default function Settings() {
                                     setCustomPronouns={setCustomPronouns}
                                 />
                                 <RolesSelection selectedRoles={selectedRoles} setSelectedRoles={setSelectedRoles} />
-
-    
-                                <SelectGenres/>
-    
-                                <button
-                                    className="bg-blue-500 hover:bg-blue-600 text-white font-bold py-2 px-4 rounded mt-2"
-                                    onClick={handleSaveChanges}
-                                >
-                                    Save Changes
-                                </button>
-                                {showSavedMessage && (
-                                    <p className="text-green-500 mt-2">Saved successfully!</p>
-                                )}
     
                                 <div className="flex flex-col mt-4"> {/* Start of username change section */}
-                                    <label htmlFor="desiredUsername" className="font-bold">Desired Username:</label>
+                                    <label htmlFor="desiredUsername" className=" text-2xl font-bold mt-4">Desired Username</label>
                                     <input
                                         type="text"
                                         id="desiredUsername"
-                                        className="border rounded p-2 mt-1"
+                                        className="border rounded p-2 mt-2 w-96"
                                         value={desiredUsername}
                                         onChange={(e) => setDesiredUsername(e.target.value)}
                                     />
                                     <button
-                                        className="bg-blue-500 hover:bg-blue-600 text-white font-bold py-2 px-4 rounded mt-2"
-                                        onClick={handleUsernameChangeRequest}
-                                    >
+                                        className="w-96 h-10 bg-maroon hover:bg-opacity-70 text-white font-bold py-2 px-10 rounded-full mt-4"
+                                        onClick={handleUsernameChangeRequest} >
                                         Request Username Change
                                     </button>
+
                                     {usernameChangeRequested && (
                                         <p className="text-green-500 mt-2">Username change requested!</p>
                                     )}
                                 </div> {/* End of username change section */}
     
                                 <button
-                                    className="bg-red-500 hover:bg-red-600 text-white font-bold py-2 px-4 rounded mt-4"
+                                    className="bg-maroon hover:bg-opacity-70 text-white font-bold py-2 px-4 rounded-full mt-4"
```

Comment:
Can we make a tailwind entry for this color, like light-maroon?

Reply:
I added the color to tailwing.config.js and changed it here.